jobs/speak-clearly-llc-python-telephony-developer.html

```diff
@@ -0,0 +1,42 @@
+---
+title: Python telephony developer
+company: Speak Clearly, LLC
+url: https://bit.ly/twiliojob
+location: Remote
+contract: perm
+contact:
+    name: Jim Salsman
+    email: jim@talknicer.com
+    ghangouts: jsalsman
+created: !!timestamp 2019-07-09
+tags: 
+  - Python	
```

Comment:
You do not need Python as a tag on the Python jobs board.

Reply:
@salimfadhley thank you! edited: https://github.com/pythonjobs/jobs/pull/385/commits/3f73286f5245dcebd10bbfc3a6d45eccf5722b6a